### PR TITLE
Ability for trying to import required libraries for certain tests

### PR DESCRIPTION
**Pull request by RukhovichIV**
this PR isn't ready yet

### CODE REVIEW

**Review by napetrov — APPROVED** [review]
Looks good. let's wait for CI

File: tests/test_examples.py [hunk 1]
```diff
@@ -45,6 +45,15 @@ def check_version(rule, target):
                     break
     return True
 
+def check_libraries(rule):
+    for rule_item in rule:
+        try:
+            import rule_item
```
Comment (PetrovKP) [inline tests/test_examples.py:51]:
```suggestion
            __import__(rule_item)
```
See [here](https://stackoverflow.com/questions/8718885/import-module-from-string-variable) info.

Comment (RukhovichIV) [reply tests/test_examples.py]: thank you, fixed it

**Comment by ShvetsKS** [comment]
seems added examples are passed on linux
https://travis-ci.com/github/IntelPython/daal4py/jobs/383932757#L1256

**Review by PetrovKP — APPROVED** [review]

**Pull request merged**